lib/operations/accessor/missingSome.js

```diff
@@ -0,0 +1,16 @@
+function missingSome(apply, need_count, options) {
+  // missing_some takes two arguments, how many (minimum) items must be present, and an array of keys (just like 'missing') to check for presence.
+  var are_missing = apply({
+    missing: options
+  }, this);
+
+  if (options.length - are_missing.length >= need_count) {
+    return [];
+  }
+
+  return are_missing;
+}
+
+missingSome.code = 'missing_some';
+missingSome.withApply = true;
```

Comment:
@lina-salih @mjenny 
Same here `code, `withApply` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014